lib/udap_security_test_kit/authorization_code_redirect_test.rb

```diff
@@ -17,7 +17,25 @@ class AuthorizationCodeRedirectTest < Inferno::Test
           title: 'Client ID',
           description: 'Client ID as registered with the authorization server.'
 
+    input :udap_authorization_code_request_scopes,
+          title: 'Scope Parameter for Authorization Request',
+          description: %(
+              A list of space-separated scopes to include in the authorization request. If included, these may be equal
+              to or a subset of the scopes requested during registration.
+              If empty, scope will be omitted as a parameter to the authorization endpoint.
+          ),
+          optional: true
+
+    input :udap_authorization_code_request_aud,
```

Comment:
Is there a reason not to just use a checkbox input or configuration option for this?

Reply:
Yes, I originally attempted a checkbox but then there would have been two inputs for it: the checkbox input to use/not use the `aud` param and then the  `udap_fhir_base_url` (originally input in the Discovery group) to get the actual value needed for the request. I thought it may be kind of confusing when running from the authorization and authentication group level (group 1.3), to have an input for the `aud` param value and then also a checkbox to opt to use it even though the value is already present, as opposed to "if you want to use this parameter, provide an input value for it; if it's empty the parameter won't be used," which felt simpler.

That said, I didn't really love any options I thought of for this input, and am open to cleaner suggestions.

Reply:
I don't think this should be a distinct input from the base url. So I think I would vote for a checkbox plus base url input.

Reply:
I updated the tests to use the checkbox and base url inputs.